doc/source/whatsnew/v1.1.0.rst

```diff
@@ -434,7 +434,8 @@ Performance improvements
 
 Bug fixes
 ~~~~~~~~~
-
+Fixed Series.value_counts so that normalize excludes NA values when dropna=False. (:issue:`25970`)
+Fixed Dataframe Groupby value_counts with bins (:issue:`32471`)
```

Comment:
can you move this down into the Groupby/resample/rolling section. starts on L596.